src/app/page.module.css

```diff
@@ -1,168 +1,153 @@
-.page {
-  --gray-rgb: 0, 0, 0;
-  --gray-alpha-200: rgba(var(--gray-rgb), 0.08);
-  --gray-alpha-100: rgba(var(--gray-rgb), 0.05);
-
-  --button-primary-hover: #383838;
-  --button-secondary-hover: #f2f2f2;
-
-  display: grid;
-  grid-template-rows: 20px 1fr 20px;
-  align-items: center;
-  justify-items: center;
-  min-height: 100svh;
-  padding: 80px;
-  gap: 64px;
-  font-family: var(--font-geist-sans);
-}
-
-@media (prefers-color-scheme: dark) {
-  .page {
-    --gray-rgb: 255, 255, 255;
-    --gray-alpha-200: rgba(var(--gray-rgb), 0.145);
-    --gray-alpha-100: rgba(var(--gray-rgb), 0.06);
-
-    --button-primary-hover: #ccc;
-    --button-secondary-hover: #1a1a1a;
-  }
-}
-
-.main {
+.backgroundImage {
+  background-image: url("/bghp.png");
+  position: absolute;
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center 30%;
+  width: 100%;
+  height: 100vh;
+  overflow: hidden;
+}
+
+.burgerButton {
+  display: none;
+}
+
+.mainContainer {
+  height: 100vh;
+  overflow: hidden;
+  width: 100%;
+  position: relative;
+  padding: 46px 0;
   display: flex;
   flex-direction: column;
-  gap: 32px;
-  grid-row-start: 2;
-}
-
-.main ol {
-  font-family: var(--font-geist-mono);
-  padding-left: 0;
-  margin: 0;
-  font-size: 14px;
-  line-height: 24px;
-  letter-spacing: -0.01em;
-  list-style-position: inside;
-}
-
-.main li:not(:last-of-type) {
-  margin-bottom: 8px;
-}
-
-.main code {
-  font-family: inherit;
-  background: var(--gray-alpha-100);
-  padding: 2px 4px;
-  border-radius: 4px;
-  font-weight: 600;
-}
-
-.ctas {
+  align-items: center;
+  justify-content: flex-start;
+}
+
+.navContainer {
+  margin: 0 auto;
+  width: 1000px;
+  max-width: 90%;
+  transform: skewX(-40deg);
+  height: 117px;
+  border: 3px solid rgb(0, 0, 0);
+  border-top-right-radius: 25px;
```

Comment:
You can refactor with border-radius only : https://developer.mozilla.org/en-US/docs/Web/CSS/border-radius